napari/_qt/dialogs/qt_reader_dialog.py

```diff
@@ -117,6 +118,7 @@ def handle_gui_reading(
     stack: Union[bool, List[List[str]]],
     plugin_name: Optional[str] = None,
     error: Optional[ReaderPluginError] = None,
+    plugin_override: Optional[bool] = False,
```

Comment:
Why optional? The default value is bool. 
```suggestion
    plugin_override: bool = False,
```

Reply:
this is still not adresed witch change or explanation.

Reply:
Ooops, my bad this one escaped me